src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -6,9 +6,6 @@
 #include "SSVOpenHexagon/Global/Assets.hpp"
 #include "SSVOpenHexagon/Utils/Utils.hpp"
 #include "SSVOpenHexagon/Core/HexagonGame.hpp"
-#include "SSVOpenHexagon/Components/CWall.hpp"
-#include "SSVOpenHexagon/Components/CCustomWallHandle.hpp"
-#include "SSVOpenHexagon/Components/CCustomWall.hpp"
```

Comment:
Why are you removing this? There are custom wall functions defined here that need these headers.

Reply:
they are not even used in the include.